packages/quantic/force-app/main/default/lwc/quanticFacet/quanticFacet.js

```diff
@@ -249,6 +262,7 @@ export default class QuanticFacet extends LightningElement {
       facetId: this.facetId ?? this.field,
       filterFacetCount: !this.noFilterFacetCount,
       injectionDepth: Number(this.injectionDepth),
+      customSort: this.customSort?.length ? [...this.customSort] : undefined,
```

Comment:
why the shallow copy here? we are not modifying the customSort array are we? just reading the data?

Reply:
It's because of the Proxy object that Salesforce uses whenever a property of a non primitive type like an array is passed  to a component, so the array will transformed to a Proxy of a an array and this latter will cause issues and errors with the Headless controller.

So to avoid that, I'm doing `[...this.customSort]` to transform a Proxy array to a regular array.